hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/DurableExecutorServiceProxy.java

```diff
@@ -180,18 +200,27 @@ public void shutdown() {
         Collection<Member> members = nodeEngine.getClusterService().getMembers();
         OperationService operationService = nodeEngine.getOperationService();
         Collection<Future> calls = new LinkedList<Future>();
+        QuorumException exception = null;
 
         for (Member member : members) {
             if (member.localMember()) {
```

Comment:
do you know why there is a special treatment for a local member?

Reply:
maybe @gurbuzali would know?

Reply:
Removed as discussed on slack.